composer.json

```diff
@@ -49,6 +50,16 @@
 		"psy/psysh": "Enhanced `wp shell` functionality"
 	},
 	"autoload": {
-		"psr-0": { "WP_CLI": "php" }
+		"psr-0": { "WP_CLI": "php" },
+		"psr-4": { "": "php/commands/src" },
+		"classmap": [ "php/export" ]
```

Comment:
I'm currently trying to figure out the autoloading in general here. There's still an issue with the `Spyc` class, my proposed change did not really help at all.